### ISSUE TITLE
our able designers

### ISSUE THREAD

**Issue by Emmahchinonso** [Emmahchinonso/Hng-login_page-project#8]
someone will help you out oladapo daniel

**Comment by Oladapodaniel** [comment]
Alright sir

**Comment by Emmahchinonso** [comment]
Lol, has your second contacted you yet

**Comment by origho** [comment]
I mentioned earlier that I can support with designs.

**Comment by Jaylukmann** [comment]
I sent Oladapo a login design, he said he will use that too as he is working bon something.

**Comment by origho** [comment]
Noted Sir.
Well done

**Comment by Oladapodaniel** [comment]
Mr origo, since you can help with design, design the homepage while I continue with the sign up and login page

**Comment by Oladapodaniel** [comment]
On the home page, the should be the logo, login icon, sign up icon, etc, as the spirit leads☺
You can also make the lion image the bg image as concluded by the rest of us in the group

**Comment by Oladapodaniel** [comment]
Is that OKAY?

**Comment by Oladapodaniel** [comment]
Sir luke man, I'll use the template you sent for the sign up page and login

**Comment by Young-Einstein10** [comment]
Is there anyway we can just see something like a wireframe or prototype of all d pages so that everyone can have a visual representation of what we're doing....we can't wait for our designers for that long also so d moment our designs are ready it'll b easy substituting our already onground codes for d designs.....

**Comment by Emmahchinonso** [comment]
Have u guys gotten the invite link to be collaborators for the projects repository

**Comment by origho** [comment]
I have gotten it

**Comment by origho** [comment]
I am sending message through my mobile phone. I could not get the mobile github

**Comment by Oladapodaniel** [comment]
![Desktop](https://user-images.githubusercontent.com/53026243/65070093-e9533b80-d983-11e9-9af8-8a9db0e48e65.png)
lets  start work with this

**Comment by Oladapodaniel** [comment]
front enders should start working on that, it just simple html and css, you could add some javascript to add some interaction

**Comment by Oladapodaniel** [comment]
origo have you started the design

**Comment by Oladapodaniel** [comment]
![Desktop (1)](https://user-images.githubusercontent.com/53026243/65074457-c711eb80-d98c-11e9-9be9-83a74e5c9924.png)
this is for the login page

**Comment by Eshjay** [comment]
I suggest we use the same background used on our sign-up page in log-in page also. To follow the rule of consistency.

**Comment by Young-Einstein10** [comment]
I agree with Eshjay....consistency is key also, we should use the same background we used for the sign up page also. Mr Origo if u are using Github from ur mobile phone, I'll advise you use Chrome then change to desktop view.

**Comment by Oladapodaniel** [comment]
![Desktop (2)](https://user-images.githubusercontent.com/53026243/65131084-8484f900-d9f6-11e9-8732-323634564b54.png)
Updated!

**Comment by Oladapodaniel** [comment]
[Working Files.zip](https://github.com/Emmahchinonso/Hng-login_page-project/files/3625239/Working.Files.zip)
These  are the images used.

**Comment by Oladapodaniel** [comment]
![Desktop (4)](https://user-images.githubusercontent.com/53026243/65136619-cbc3b780-d9ff-11e9-8182-6fa8fbca75d1.png)
Sign Up Page Updated!

**Comment by Oladapodaniel** [comment]
![iPad Pro 12 9_](https://user-images.githubusercontent.com/53026243/65151479-1b18e080-da1e-11e9-8af2-cdb8a1da0b7d.png)
Check Out Our Home Page.

**Comment by Young-Einstein10** [comment]
in my opinion i feel all these designs are not needed, the reason why we are creating homepage is just to be able to link to our signup page and login page, besides imagine the time frontend will use also in implementing all these designs, time is not on our side. Instead, we should just have like three contents in our homepage, sign up and login at the top right hand corner and a welcome message at the center.

**Comment by Emmahchinonso** [comment]
yea supported, will you be making any more changes to the login page

**Comment by Oladapodaniel** [comment]
The other links should just be dead links, except the sign up and login link which will link the other pages, if you feel they are still not needed, redesign it yourself, I'll send you the link
